denops/gin/command/edit/write.ts

```diff
@@ -65,8 +65,11 @@ export async function exec(
     restore = () => Deno.remove(original);
   }
   try {
+    const stat = await Deno.stat(f);
     await fs.ensureFile(original);
-    await Deno.writeTextFile(original, `${content.join("\n")}\n`);
+    await Deno.writeTextFile(original, `${content.join("\n")}\n`, {
+      mode: stat.mode ?? undefined,
```

Comment:
So that `Deno.writeTextFile` **change** file mode even the `mode` attribute is not specified? I think it's a kind of issue on the Deno API.